caps/types.go

```diff
@@ -85,6 +113,18 @@ func (t *BoolFileType) SecuritySnippet(c *Capability, securitySystem SecuritySys
 	}
 }
 
+func (t *BoolFileType) dereferencedPath(c *Capability) (string, error) {
+	path := c.Attrs["path"]
+	if path == "" {
+		return "", &notSanitizedError{}
```

Comment:
return fmt.Errorf("bool-file capability is invalid: missing path attribute")

?